packages/flutter/lib/src/material/slider.dart

```diff
@@ -474,13 +474,22 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
   Timer? interactionTimer;
 
   final GlobalKey _renderObjectKey = GlobalKey();
+
   // Keyboard mapping for a focused slider.
-  final Map<ShortcutActivator, Intent> _shortcutMap = const <ShortcutActivator, Intent>{
+  final Map<ShortcutActivator, Intent> _traditionalNavShortcutMap = const <ShortcutActivator, Intent>{
       SingleActivator(LogicalKeyboardKey.arrowUp): _AdjustSliderIntent.up(),
```

Comment:
Are we sure that these are even correct.  I mean, it's convenient to have up and down change the slider, but are they actually idioms that anyone knows about?  I can see it being important for a vertical slider, but we don't have those.

Because if they're not really idiomatic, then we can just eliminate them and directional nav will just work.

Reply:
I don't know the history behind that particular bit of functionality so I went with the solution affected the least amount of things.  I'd personally be fine with that smaller change if you think it's safe.